docs/source/index.rst

```diff
@@ -147,9 +156,9 @@ Indices and tables
 
 Questions? Suggestions?
 =======================
+All questions and suggestions are welcome. Please feel free to use our `Jupyter mailing list <https://groups.google.com/forum/#!forum/jupyter>`_ and `Jupyter website <https://jupyter.org>`_ to contact our team.
```

Comment:
The website doesn't help contact the team, so maybe it doesn't belong in this section.

While we're here, can you replace the mailing list link with the forum: https://discourse.jupyter.org ?